alembic/versions/0003_response_event.py

```
@@ -46,6 +46,10 @@
 $$ LANGUAGE plpgsql;
 """
 
+# 治理裁决（#43 Minor 项）：CodeRabbit 建议 FOR EACH ROW，但语句级语义是
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)